config/locales/dn.yml

```diff
@@ -0,0 +1,28 @@
+# Danish
+dn:
```

Comment:
For Danish translations, it should use the Danish language code which is `da` (not `dn`).
Could you change this and the filename accordingly?

Reply:
I change the name of the file and the accronyme in the file